edb/pgsql/compiler/dml.py

```diff
@@ -437,6 +471,27 @@ def process_insert_body(
                             path_id=ictx.rel.path_id, ctx=ctx)
         iterator_id = pathctx.get_path_identity_var(
             select, iterator_set.path_id, env=ctx.env)
+
+        # debug.header("IT IS THAT ITERATOR SHIT")
```

Comment:
Let's not commit any debug code, even when it's commented out

Reply:
Oh, crap.